components/webgpu/wgpu_thread.rs

```diff
@@ -86,8 +89,11 @@ pub(crate) struct WGPU {
     /// Provides access to poller thread
     poller: Poller,
     /// Store compute passes (that have not ended yet) and their validity
+    // we need to store valid field, because wgpu does not invalidate pass on error
     compute_passes: HashMap<ComputePassId, (Box<dyn DynComputePass>, bool)>,
-    //render_passes: HashMap<RenderPassId, Box<dyn DynRenderPass>>,
+    /// Store compute passes (that have not ended yet) and their validity
+    // we need to store valid field, because wgpu does not invalidate pass on error
+    render_passes: HashMap<RenderPassId, (Box<dyn DynRenderPass>, bool)>,
```

Comment:
> use custom enum

That sounds better indeed, also that way `valid` can be a field of only the variant that has not ended?

Reply:
Yes